sympy/matrices/matrices.py

```diff
@@ -392,6 +392,7 @@ def det(self, method="bareiss"):
         """
 
         # sanitize `method`
+        method = kwargs.get('method', "bareiss")
```

Comment:
Should I remove this line also then ? @normalhuman , beacuse the kwargs are of no special use here.